src/wwwroot/partials/settings/dkim-configuration-help.html

```diff
@@ -10,9 +10,7 @@
   <hr class="default" />
 </div>
 <div class="item section--wrapper flex">
-  <h2>
-    {{ 'dkim_configuration' | translate }}
-  </h2>
+  <h3><strong>{{ 'dkim_configuration' | translate }}</strong></h3>
```

Comment:
Yeah, we should not add a bold into any h3 header, we have several others h3 in the app and no one have bold font weight, so, lets have a small meeting so we can work this out.